arm-eventhub/2017-04-01/examples/NameSpaces/EHNameSpaceCheckNameAvailability.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "api-version": "2017-04-01",
+    "subscriptionId": "5f750a97-50d9-4e36-8081-c9ee4c0210d4",
+    "parameters": {
+      "name": "sdk-Namespace-8458"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "nameAvailable": true,
+        "reason": "None",
+        "message": ""
+      }
```

Comment:
1. how about a negative example where the name is unavailable?
2. similarly in other examples, try to incorporate a wide range of response scenarios instead of just the positive scenario